README_BUILD.md

```diff
@@ -134,6 +134,15 @@ analog@precision:~/libiio/build$ make -j$(nproc)
 analog@precision:~/libiio/build$ sudo make install
 ```
 
+Note: As specified above, the default installation path on Linux based systems is '\usr'.
```

Comment:
Looks good. Just a tiny remark: it should be '/usr' instead of '\usr'.

Reply:
fixed